### PR TITLE
Previous wins populating

### PR DESCRIPTION
**Pull request by jaleszek**
The "Previous Winnings" section needs to pre-populate from the Application form (but also be editable)

### CODE REVIEW

File: app/services/form_answer_user_submission_service.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  end
+
+  def perform
+    assign_previous_wins
+    Notifiers::Submission::SuccessNotifier.new(form_answer).run
+  end
+
+  private
+
+  def assign_previous_wins
```
Comment (matthewford) [inline app/services/form_answer_user_submission_service.rb]: Assignment Branch Condition size for assign_previous_wins is too high. [15.43/15]

File: app/services/form_answer_user_submission_service.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return unless form_answer.previous_wins.blank?
+
+    award_holder_details = form_answer.document["queen_award_holder_details"]
+    if award_holder_details.present?
+      json = JSON.parse(award_holder_details)
+      json.each do |win|
+        win = JSON.parse(win)
+        form_answer.previous_wins.create(category: win["category"], year: win["year"] )
```
Comment (matthewford) [inline app/services/form_answer_user_submission_service.rb]: Space inside parentheses detected.

File: spec/services/form_answer_user_submission_service_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+describe FormAnswerUserSubmissionService do
+  subject { described_class.new(form_answer) }
+
+  let(:document) do
+    {
+      "queen_award_holder_details" => "[\"{\\\"category\\\":\\\"innovation\\\",\\\"year\\\":\\\"2012\\\"}\", \"{\\\"category\\\":\\\"innovation\\\",\\\"year\\\":\\\"2011\\\"}\", \"{\\\"category\\\":\\\"international_trade\\\",\\\"year\\\":\\\"2015\\\"}\", \"{\\\"category\\\":\\\"sustainable_development\\\",\\\"year\\\":\\\"2014\\\"}\", \"{\\\"category\\\":\\\"sustainable_development\\\",\\\"year\\\":\\\"\\\"}\", \"{\\\"category\\\":\\\"\\\",\\\"year\\\":\\\"\\\"}\"]"
```
Comment (matthewford) [inline spec/services/form_answer_user_submission_service_spec.rb]: Line is too long. [471/100]

**Comment by RuslanHamidullin** [comment]
Good :+1:

**Pull request merged**